### ISSUE TITLE
Scrollbar is sometimes stuck in scrolling

### ISSUE THREAD

**Issue by btlechowski** [brave/browser-laptop#13580]
### Steps to Reproduce
1. Open: about:welcome
2. Move mouse pointer over scrollbar and hold it with Left Mouse Button
3. While holding Left Mouse Button move the pointer over URL field
4. Release Left Mouse Button

**Actual result:**
![scroll broken](https://user-images.githubusercontent.com/34715963/37836035-cee97ad0-2eb1-11e8-9081-554113cccaae.gif)

**Expected result:**
Scrollbar is not stuck in scrolling

**Reproduces how often:**
100%

### Brave Version
```
Brave: 0.22.7 
V8: 6.5.254.41 
rev: 8bb7e77f69bca186429aafda4bc6ba4d82a5de47 
Muon: 5.1.1 
OS Release: 6.1.7601 
Update Channel: Beta 
OS Architecture: x64 
OS Platform: Microsoft Windows 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 65.0.3325.181
```
@LaurenWags reported that it is reproducible on v0.21.24, but not on v0.21.18

### Additional Information
Also reproduced when moving mouse pointer outside of Brave window. 
Below reproduced for scrollbar arrow:
![scroll arrow broken](https://user-images.githubusercontent.com/34715963/37836192-2ce1b99a-2eb2-11e8-82c0-7499caf84f38.gif)

cc @LaurenWags @srirambv @kjozwiak @bsclifton

**Comment by LaurenWags** [comment]
many +1s from community: 
https://community.brave.com/t/cant-stop-vertical-scrolling/18194
https://community.brave.com/t/scroll-bar-wont-stop-scrolling/18135

**Comment by jayyyin** [comment]
has this issue been fixed?

**Comment by kjozwiak** [comment]
> has this issue been fixed?

Looks like @darkdh has been working on a fix via https://github.com/brave/muon/commit/554772bfab1aceb7e719a9e5d74cd22ce2e2b528. The PR is on the muon side of things. Once the new muon version with the above fix is merged into `0.22.x`, this will be closed and then verified by QA.

**Comment by btlechowski** [comment]
Verified on Windows 7 x64
* 0.22.8    3ae27f2
* libchromiumcontent: 65.0.3325.181
* muon: 5.1.2

**Comment by btlechowski** [comment]
**The issue still reproduces for a thin line of pixels close to window frame.**
1. Open: about:welcome
2. Move mouse pointer over center of the scrollbar and hold it with Left Mouse Button
3. While holding Left Mouse Button move the pointer close to window frame on the right
4. Release Left Mouse Button

**Actual result:**
![scrollbar_window_frame](https://user-images.githubusercontent.com/34715963/38273391-1b331882-378c-11e8-840e-af6ded2c01b9.gif)

Also reproducible for scrollbar arrows

**Brave info**
```
Brave: 0.22.13 
V8: 6.5.254.41 
rev: a8cfb160479f1d00d0769368eb440030182bb83b 
Muon: 5.1.2 
OS Release: 6.1.7601 
Update Channel: Release 
OS Architecture: x64 
OS Platform: Microsoft Windows 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 65.0.3325.181
```

**Comment by bsclifton** [comment]
removed OS labels since this affects all platforms

**Comment by darkdh** [comment]
window frame on the right issue seems like Win 7 only. I can't reproduce it on Win 10 and other platforms

**Comment by LaurenWags** [comment]
I can't say it's this particular scrolling issue (@btlechowski logged several scenarios for scrolling problems) but there are multiple reports of scrolling issues on Win 10 in community. Some of which are mentioned here https://github.com/brave/browser-laptop/issues/13580#issuecomment-375696324

**Comment by LaurenWags** [comment]
Multiple +1s from community:
https://community.brave.com/t/scroll-bar-wont-stop-scrolling/18135
https://community.brave.com/t/cant-stop-vertical-scrolling/18194

**Comment by bsclifton** [comment]
This issue was closed as `wontfix` since Brave Core does not have this issue and we'll be moving over to that soon (from our current Muon based product). More information about Brave Core available here:
https://brave.com/development-plans-for-upcoming-release/
